### PR TITLE
Submit Week3-- Salih S.

### PR DESCRIPTION
**Pull request by salih18**
Week- 3   https://naughty-galileo-a7b4b3.netlify.com/

Week-2    https://angry-brattain-c60e5e.netlify.com

# Component Diagram

components

### CODE REVIEW

File: homework-submission/react-hyf-client/src/components/AppContainer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import Navbar from '../components/Navbar';
+import UserCard from '../components/UserCard';
+import useKeyPress from '../hooks/useKeyPress';
+
+function AppContainer() {
+  const keyPressData = useKeyPress();
+  return (
+    <div className="AppContainer">
+      <Navbar />
+      {keyPressData === 0 && (
```
[the remark this answers is not in the document]
Comment (salih18) [reply homework-submission/react-hyf-client/src/components/AppContainer.js]: Yes you are right.. i will change it.

File: homework-submission/react-hyf-client/src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+import './Navbar.css';
+import NavbarButton from './NavbarButton';
+import { NavContext } from '../contexts/UserContext';
+import { randomURL } from '../contexts/UserContext';
```
[the remark this answers is not in the document]
Comment (salih18) [reply homework-submission/react-hyf-client/src/components/Navbar.js]: I didnt see it :)

**Comment by salih18** [comment]
Thank you @ArcoMul  for your review... I will implement your suggestion..

**Comment by salih18** [comment]
>Besides that I noticed that when I press space bar the page scrolls down a bit, that's default browser behaviour, can you try to prevent that?

@ArcoMul  I have tried both firefox and chrome in Ubuntu,  no scrolling however sometimes scrolls because of the size of the content in UserCard component...

**Comment by ArcoMul** [comment]
No need to close the pull request @salih18, changes can be pushed to the branch of this pull request to appear here.

**Comment by salih18** [comment]
> No need to close the pull request @salih18, changes can be pushed to the branch of this pull request to appear here.

Sorry, I did close by accidentally :/

**Pull request closed without merging**